tests/test_build.py

```diff
@@ -179,6 +181,33 @@ def test_no_base_image(tmpdir):
     assert "no base image specified" in str(exc.value)
 
 
+def test_copy_from_is_blocked(tmpdir):
+    """test when user has specified COPY --from=image (instead of builder)"""
+    dfp = df_parser(str(tmpdir))
+    if MOCK:
+        mock_docker()
+    source = {'provider': 'path', 'uri': 'file://' + str(tmpdir), 'tmpdir': str(tmpdir)}
+
+    dfp.content = dedent("""\
+        FROM monty AS vikings
+        FROM python
+        COPY --from=vikings /spam/eggs /bin/eggs
+        COPY --from=0 /spam/eggs /bin/eggs
+        COPY src dest
+    """)
+    InsideBuilder(get_source_instance_for(source), 'built-img')  # calls set_df_path at init
+    # should not raise an error
+
+    dfp.content = dedent("""\
+        FROM monty as vikings
+        FROM python
+        COPY --from=notvikings /spam/eggs /bin/eggs
+    """)
+    with pytest.raises(RuntimeError) as exc_info:
+        InsideBuilder(get_source_instance_for(source), 'built-img')  # calls set_df_path at init
+    assert "COPY --from" in str(exc_info.value)
```

Comment:
is it worth adding one more test case like this?:
```
dfp.content = dedent("""\
        FROM monty as vikings
        FROM python
        COPY --from=2 /spam/eggs /bin/eggs
    """)
```

It's the same logic as notvikings, I get that, but it might be useful for someone puzzling out the code in 6 months.

Reply:
yes, thanks!